modules/ztp-image-based-upgrade-with-rhacm.adoc

```diff
@@ -0,0 +1,362 @@
+If the `lca.openshift.io/apply-label` annotation is used in the `Backup` CR, only the resources listed in the annotation will be backed up, even if other resource types are specified in the `spec` section or not.
+====
+--
+
+.. Define the apply order for the OADP Operator in the `Restore` CRs by using the `lca.openshift.io/apply-wave` field:
++
+--
+.Example OADP CRs
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: backup-example-app
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - test
+  includedNamespaceScopedResources:
+  - secrets
+  - persistentvolumeclaims
+  - deployments
+  - statefulsets
+  excludedClusterScopedResources:
+  - persistentVolumes
+----
+
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-acm-klusterlet
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "1"
+spec:
+  backupName:
+    backup-acm-klusterlet
+----
+
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-example-app
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "2"
+spec:
+  backupName:
+    backup-example-app
+----
+
+[NOTE]
+====
+If you do not define the `lca.openshift.io/apply-wave` annotation in the `Backup` or `Restore` CRs, they will be applied together.
+====
+--
+
+. Generate a `ConfigMap` object for your OADP CRs.
+
+.. Create the `ConfigMap` object:
++
+[source,terminal]
+----
+$ oc create configmap example-oadp-cm --from-file=example-oadp-resources.yaml=<path_to_oadp_crs> -n openshift-adp
+----
+
+.. Apply the `ConfigMap`:
++
+[source,terminal]
+----
+$ oc apply -f oadp-cm-example.yaml
+----
+
+. (Optional) Create `ConfigMap` resources that contain the additional manifests that you want to apply to the target cluster.
+
+.. Create a YAML file that contains your extra manifestss.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: "pci-sriov-net-e5l"
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+    pfNames: [ens1f0]
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  mtu: 1500
+  numVfs: 8
+  priority: 99
+  resourceName: pci_sriov_net_e5l
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: "networking-e5l"
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: |-
+    {
+    }
+  linkState: auto
+  networkNamespace: flexran-testcases
+  resourceName: pci_sriov_net_e5l
+  spoofChk: "on"
+  trust: "off"
+----
+
+.. Create the `ConfigMap` object:
++
+[source,terminal]
+----
+$ oc create configmap example-extra-manifests-cm --from-file=example-extra-manifests.yaml=<path_to_extramanifest> -n openshift-lifecycle-agent
```

Comment:
Looking at "--from-file=example-extra-manifests.yaml" 
Do we want to clarify that the name example-extra-manifests.yaml is arbitrary ?

Reply:
See my other comment on example-catalogsources.yaml